src/Url.js

```diff
@@ -52,7 +52,7 @@ const Url = React.createClass({
 
   onClick(event) {
     event.preventDefault();
-    ActionCreator.navigateTo(this.props);
+    Store.dispatch(ActionCreator.navigateTo(this.props));
```

Comment:
I think it is okay for now. No need to change this part. Lib is too small to add one more layer here. We can also try that later.